app/models/reading_collection.rb

```diff
@@ -4,4 +4,12 @@ class ReadingCollection < ApplicationRecord
   has_many :articles, through: :reading_collection_articles
 
   validates :name, :user_id, presence: true
+  acts_as_taggable_on :tags
+
+  def get_articles
+    Article.where("created_at >= ?", 1.week.ago).
+      tagged_with(tag_list, any: true).
+      order("page_views_count DESC").
+      limit(10)
```

Comment:
Looks like from there it's doing ActiveRecord stuff on Article using that tagged_with gem you mentioned which is neat. Looks like a limit of 10 is in place right now, that meaning only ten results at a time currently?

Reply:
Correct! It can be changed to whatever we want.

Reply:
Yea that'll populate 10 articles based on the whole search. However, the user can make more collections. 

We can bump that number as we see fit.